### PR TITLE
Alumni Page Updated

### PR DESCRIPTION
**Pull request by RijulBansal**

### CODE REVIEW

File: src/Pages/Alumni/Components/Sampleimg.png
Comment (nimishmedatwal) [inline src/Pages/Alumni/Components/Sampleimg.png:1]: Remove this pic

File: src/Pages/Alumni/Assets/alumni.jpg
Comment (nimishmedatwal) [inline src/Pages/Alumni/Assets/alumni.jpg:1]: Remove this sample image

File: src/Pages/Alumni/Main.js [hunk 1]
```diff
@@ -0,0 +1,338 @@
+// import React, { useState } from "react";
```
Comment (nimishmedatwal) [inline src/Pages/Alumni/Main.js:1]: Delete all the commented code.

File: src/Pages/Alumni/PolaroidGrid.js [hunk 2]
```diff
@@ -0,0 +1,316 @@
+// import React, { useEffect, useState } from "react";
```
Comment (nimishmedatwal) [inline src/Pages/Alumni/PolaroidGrid.js:1]: Remove the commented code here also

File: src/Pages/Alumni/Styles/Style.module.css [hunk 3]
```diff
@@ -0,0 +1,274 @@
+/* @import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
```
Comment (nimishmedatwal) [inline src/Pages/Alumni/Styles/Style.module.css:1]: .

File: src/Pages/Alumni/Styles/PolaroidGrid.module.css [hunk 4]
```diff
@@ -0,0 +1,386 @@
...
+  width: 33.1vh;
+} */
+
+.items{
+  width: 80vh;
+  height:60vh;
+  /* margin-left: 6rem; */
+}
+
+/* .image {
```
Comment (nimishmedatwal) [inline src/Pages/Alumni/Styles/PolaroidGrid.module.css:240]: .

File: src/Pages/Alumni/Styles/PolaroidGrid.module.css [hunk 5]
```diff
@@ -0,0 +1,386 @@
...
+  /* transition: 0.5s; */
+}
+
+/* .productCard__content:hover.productName:after{
+  opacity:50%;
+} */
+.productInfo{
+  visibility: hidden;
+  font-size: 1.2rem;
+  /* opacity: 100%;  */
```
Comment (nimishmedatwal) [inline src/Pages/Alumni/Styles/PolaroidGrid.module.css:214]: .

File: src/Pages/Alumni/Styles/Years.module.css [hunk 6]
```diff
@@ -0,0 +1,156 @@
+/* @import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
```
Comment (nimishmedatwal) [inline src/Pages/Alumni/Styles/Years.module.css:1]: .

**Review by nimishmedatwal — COMMENTED** [review]
Important changes:
- Change app.js in src so that your page can be rendered.
- Change the Navbar as well to show Alumini button
Minor changes: 
-  remove **all** the code that is commented also remove the two sample images.

File: src/Pages/Alumni/Styles/Card.module.css [hunk 7]
```diff
@@ -0,0 +1,69 @@
+
+:root {
+  --color-grey-primary: #f0f0f0;
+  --color-yellow-primary: goldenrod;   
+  --color-orange-primary: darkorange;
+  --color-green-primary: mediumseagreen;
+}
+ 
+html{
```
Comment (MankiratSingh1315) [inline src/Pages/Alumni/Styles/Card.module.css:9]: Don't apply font styling on html as it changes the styling of other pages too.

**Review by MankiratSingh1315 — CHANGES_REQUESTED** [review]

**Comment by MankiratSingh1315** [comment]
I have updated the images of 2022 alumni. Please make them fit in the carousel without shrinking.

**Pull request merged**